app/manage-projects/[projectSlug]/contributions/_features/kanban-view/kanban-view.tsx

```diff
@@ -71,7 +72,7 @@ function Column({
       isFetchingNextPage={isFetchingNextPage}
       header={{
         title,
-        badge: { children: data?.pages?.[0]?.totalItemNumber ?? "0" },
+        badge: { children: count, count },
```

Comment:
Use count instead of children?

Reply:
don't think we want to overide the children props with this one ?